go/goroute/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.12.8 AS buildenv
+
+ENV GO111MODULE=on
```

Comment:
@waghanza I also added GO111MODULE=on which is missing in almost all go web frameworks. If this variable is not set and working dir is set to /go/src/app go will treat this as old GOPATH build setup and will not use go modules correctly.